tdns/queryresponder.go

```
@@ -43,7 +44,7 @@ var standardDNSTypes = map[uint16]bool{
 	dns.TypeCDNSKEY:    true,
 }
 
-func (zd *ZoneData) ApexResponder(w dns.ResponseWriter, r *dns.Msg, qname string, qtype uint16, dnssec_ok bool, kdb *KeyDB) error {
+func (zd *ZoneData) xxxApexResponder(w dns.ResponseWriter, r *dns.Msg, qname string, qtype uint16, dnssec_ok bool, kdb *KeyDB) error {
```

golangci-lint (1.62.2)

func `(*ZoneData).xxxApexResponder` is unused

(unused)